README.md

```diff
@@ -24,6 +24,24 @@ One could also use a standard Clang installation, build a sysroot from the
 sources mentioned above, and compile with
 "--target=wasm32-wasi --sysroot=/path/to/sysroot".
 
+## Install
+
+A typical installation from the release binaries might look like the following:
+```shell script
+wget https://github.com/CraneStation/wasi-sdk/releases/download/wasi-sdk-[version]/wasi-sdk-[version]-linux.tar.gz
+tar xvf wasi-sdk-[version]-linux.tar.gz
+sudo cp -r wasi-sdk-[version] /opt/wasi-sdk
```

Comment:
I would hope this `sudo` install stop is not needed.  

Maybe easier to suggest `--sysroot` argument?    At least until er can fix https://github.com/WebAssembly/wasi-sdk/issues/58

Reply:
Yeah, the sudo can go away. What would it take to fix #58?

Reply:
https://reviews.llvm.org/D76653.  But even then we would need to wait for llvm 11 to be stable before we can rely on it.

Reply:
Ah, ok. Yeah, let me change this to the more explicit version with --sysroot.